pyk4a/capture.py

```diff
@@ -50,6 +53,24 @@ def color_timestamp_usec(self) -> int:
             self.color
         return self._color_timestamp_usec
 
+    @property
+    def color_exposure_usec(self) -> int:
+        if self._color_exposure_usec is None:
+            value = k4a_module.color_image_get_exposure_usec(self._capture_handle)
+            if value == 0:
+                raise K4AException("Cannot read exposure from color image")
+            self._color_exposure_usec = value
+        return self._color_exposure_usec
+
+    @property
+    def color_white_balance(self) -> int:
```

Comment:
maybe `color_whitebalance` instead `color_white_balance` will be better?

Reply:
I prefer white_balance since it is what the SDK uses

Reply:
ok, lets's leave as is